blazor/gantt-chart/row-selection.md

```diff
@@ -221,6 +221,51 @@ You can select multiple rows by setting the `SelectionSettings.Type` property to
 
 ![Multiple Row Selection in Blazor Gantt Chart](images/blazor-gantt-chart-multi-row-selection.png)
 
+## Hover row selection
```

Comment:
this topic need to be add under Rows as like Row height add one more topic called Row Hover. It is not a right place to add under selection topic

Reply:
Changed topic under Row